Demo App/demo_app/demo_app/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
@antriksh123 You have created 3 directories, we don't need these much directories.
Just create a directory named `example` then cd into that directory.
Then run, `django-admin startproject demo .`(**don't forget to add dot(.) after demo otherwise you'll create another directory.**)
This will create minimum required folders.

Reply:
I have added the db.sqlite3 in gitignore